pkg/api/v1alpha1/cloudstackmachineconfig.go

```diff
@@ -9,7 +9,10 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-const CloudStackMachineConfigKind = "CloudStackMachineConfig"
+const (
+	DefaultCloudStackUser       = "capc"                    // DefaultCloudStackUser is the default CloudStackMachingConfig username
```

Comment:
I moved it here because the previous package at `internal/package/api` seems to be primarily geared towards constructs and methods used in the testing modules. While here contains related code for the `CloudStackMachineConfig` in the main modules.